source/includes/harvest/_introduction.md

```diff
@@ -171,6 +171,7 @@ The timestamps below are Eastern Time.
 
 | Date                    | Description                                                                                                                       |
 | ----------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| Apr 12, 2020 09:00:00AM | V1 of [PATCH: Update Job Post] has been deprecated. Updating a job post's status has been extracted to [PATCH: Update Job Post Status], while updating all other properties remain on [PATCH: Update Job Post]. The primary change is the On-Behalf-Of user requires different access permission to the job post. This more accurately reflects the required access permissions in Greenhouse Recruiting. Users should transition to the V2 endpoint.
```

Comment:
Should be "while updating all other properties **remains** on..."
I don't understand this sentence: "The primary change is the On-Behalf-Of user requires different access permission to the job post."